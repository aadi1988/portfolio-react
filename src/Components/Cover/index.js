import React from 'react';
import '../../index.css';

// background image on cover page
const Cover = () => {
  return (
      <div className="bg"></div>
  );
}

export default Cover;