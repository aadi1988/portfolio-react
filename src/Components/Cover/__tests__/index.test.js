import React from 'react'
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Cover from '../'

afterEach(cleanup)

describe('Cover is rendering', () => {
  it('renders', () => {
    render(<Cover />);
  });

  it('renders', () => {
    const { asFragment } = render(<Cover />)
    expect(asFragment()).toMatchSnapshot()
  });
});