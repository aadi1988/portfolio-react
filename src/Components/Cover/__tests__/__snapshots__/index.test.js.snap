// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cover is rendering renders 1`] = `
<DocumentFragment>
  <div
    class="bg"
  />
</DocumentFragment>
`;
